internal/internal.go

```diff
@@ -117,6 +119,8 @@ func New(m *plugins.Manager, cfg *Config) plugins.Plugin {
 		server: grpc.NewServer(
 			grpc.MaxRecvMsgSize(cfg.GRPCMaxRecvMsgSize),
 			grpc.MaxSendMsgSize(cfg.GRPCMaxSendMsgSize),
+			grpc.UnaryInterceptor(otelgrpc.UnaryServerInterceptor()),
```

Comment:
Will these automatically pick up the right configuration? 🤔

Reply:
Because this line https://github.com/vinhph0906/opa-envoy-plugin/blob/%234290/vendor/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc/grpctrace.go#L51 will get `TracerProvider` from `go.opentelemetry.io/otel/internal/global`. Then we can set global `TracerProvider` at https://github.com/vinhph0906/opa-envoy-plugin/blob/%234290/vendor/github.com/open-policy-agent/opa/internal/distributedtracing/distributedtracing.go#L119. It will get the right configuration.

Reply:
OK thanks for reminding me 😅 You're right! 👍